src/Plugins/ComplexCore/src/ComplexCore/Filters/InterpolatePointCloudToRegularGridFilter.cpp

```diff
@@ -100,6 +101,8 @@ void mapPointCloudDataByKernel(IDataArray* source, INeighborList* dynamic, std::
   case DataType::float64:
     mapPointCloudDataByKernel<float64>(source, dynamic, kernelVals, kernel, dims, curX, curY, curZ, vertIdx);
     break;
+  case DataType::boolean:
```

Comment:
It looks like preflight just skips bool arrays which is why they make it to execute. Should selecting bool arrays just be a preflight error?

Reply:
I'm down deep into those pipelines trying to figure out what to do. i'm not really sure how one "interpolates" boolean values so this should indeed be a preflight error if someone selects a bool array to interpolate

Reply:
I removed this fix because the entire InterpolatePointCloudToRegularGrid needs to be fixed. There are a bunch of issues with the filter.